openmetadata-docs/content/quick-start/local-deployment.md

```diff
@@ -347,3 +347,20 @@ WARNING! This will remove all custom networks not used by at least one container
 ```
 
 So be careful if you want to keep up some (unused) networks from your laptop.
+
+### Error while Restarting the Containers
```

Comment:
Let's remove this part as we have this fix addressed [here](https://github.com/open-metadata/OpenMetadata/issues/7493).

Reply:
Sure